.editorconfig

```diff
@@ -0,0 +1,16 @@
+#root = true
```

Comment:
Please remove this. I prefer to not have unrelated changes in PRs. Feel free to make another PR if you want to add this.